.github/workflows/test-full-sync.yml

```diff
@@ -203,7 +203,7 @@ jobs:
           --container-restart-policy=never \
           --container-stdin \
           --container-tty \
-          --container-env=ZEBRA_SKIP_IPV6_TESTS=1,TEST_FULL_SYNC=1,ZEBRA_FORCE_USE_COLOR=1,FULL_SYNC_MAINNET_TIMEOUT_MINUTES=600 \
```

Comment:
> This is probably causing some parsing errors in the full sync at some point in the logs

The most likely scenario is an intermittent and temporary network interruption, which would be fixed by the reconnection in PR #4198.

But thinking through the other possibilities...

If it was a parsing error, we'd expect it to happen at the same time in every sync. But before yesterday, the error was only happening on some syncs.

It is also possible (but unlikely) that there is a specific log message that's causing the disconnection. That would be a Google Cloud, Docker, or GitHub actions bug. And we'd expect other projects to see it as well.

So that leaves an error or panic in the container. But in that case, #4198 would not fix the problem. (And I'd expect it to also be visible at some other level of the technology stack.)